### DISCUSSION TITLE
Customize RustDesk for Windows installation

### DISCUSSION THREAD

**Discussion by steintecnologia** [rustdesk/rustdesk#8163] [Q&A]
Good morning everybody,

I need to create a custom RustDesk installer, where it already makes the following inclusions during installation;
-Set password for remote access
-Register the Server ID
-Register the Relay Server
-Register the API Server
-Register the Key

I need to install RustDesk on 60 computers running Windows 10/11 and on these computers multiple users log into the same computer during the day and I need the same settings to apply to all logins.

I don't understand anything about programming, if anyone can help, I would be grateful.

Sorry for my English, I'm using Google Translator.

**Comment by DMSnz** [comment]
You can push out such settings as a config, loaded by RustDesk installer at install time.
Look at RustDesk powershell install script.
Once you have that sorted, you can deploy via whatever means, but it needs to run as system user (so machine level GPO for example) so install service and run for all users and so on.
